src/default-test/java/nl/tudelft/jpacman/npc/ghost/NavigationTest.java

```diff
@@ -43,8 +45,10 @@
     @BeforeEach
     void setUp() {
         PacManSprites sprites = new PacManSprites();
-        parser = new MapParser(new LevelFactory(sprites, new GhostFactory(
-            sprites)), new BoardFactory(sprites));
+        LevelFactory lfact = new LevelFactory(sprites,
+            new GhostFactory(sprites),
+            Mockito.mock(PointCalculator.class));
```

Comment:
This can simply be `mock(PointCalculator.class)` (the correct import is already on line 3 ;) )

Reply:
Sorry, habit. I prefer "Mockito.mock()", but can't really explain why.

Just fixed it!